src/SFA.DAS.EmployerIncentives.Domain/ValueObjects/IncentivesConfiguration.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using SFA.DAS.EmployerIncentives.Abstractions.Domain;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.Exceptions;
+using SFA.DAS.EmployerIncentives.Enums;
+
+namespace SFA.DAS.EmployerIncentives.Domain.ValueObjects
+{
+    public class IncentivesConfiguration
```

Comment:
Configuration is not a concern of the domain.  This is not a business object.   The IncentivePaymentProfile is the business object and already exists.  No need to add this object.

Reply:
Encapsulation, payment profile selection logic doesn't need to sit inside Incentive object. It is not its concern to filter our irrelevant payment configurations, don't you think? 🤷‍♂️

Reply:
happy to rename this to make more sense?

Reply:
It's not needed.  We have the IncentivePaymentProfile that encapsulates the structure and the Incentive object that uses that to provide business functionality.  The concern of the incentive object is to use and interpret the payment profiles to calculate the payments, eligibility and phase etc...   So this object is redundant since the business functionality is already encapsulated in the incentive object.   This is not adding any Business value so should be removed and the "helper" methods moved to the incentive object.